R/loadFromMetaphlan.R

```diff
@@ -137,6 +137,8 @@ loadFromMetaphlan <- function(
             altExp(tse, rank) <- se_objects[[rank]]
         }
     }
+    # Set taxonomy ranks using .set_taxonomy_ranks
+    .set_taxonomy_ranks(tse, ranks = NULL, set.ranks = TRUE, ...)
```

Comment:
Now, user cannot decide if she/he wants to set ranks or not because this is hard-coded.

Reply:
To clarify the structure of the function.

1. set.ranks FALSE by default
2. If set.ranks == FALSE, skip the function by returning NULL
3. Check that rowData exists. If not found, give warning and skip function.
4. Get all character values from rowData. If none found, give warning and skip
5. Set ranks and give message.

And always return() something, for example, NULL. R functions return always something, so it is good to specify returned value even though it does not mean anything